xtask/src/qemu.rs

```diff
@@ -456,6 +456,10 @@ pub fn run_qemu(arch: UefiArch, opt: &QemuOpt) -> Result<()> {
     add_pflash_args(&mut cmd, &ovmf_paths.code, PflashMode::ReadOnly);
     add_pflash_args(&mut cmd, &ovmf_vars, PflashMode::ReadWrite);
 
+    // Configure SCSI Controller
+    cmd.arg("-device");
+    cmd.arg("virtio-scsi-pci,id=scsi");
```

Comment:
What is this `id=scsi` here used for? Do we need the virtio-sci-pci device at all if we add a -device sci-hd later?

Reply:
The `id` is not necessarily required, but the `virtio-scsi-pci` device is required. That is the virtual `SCSI Controller` that the disk (`scsi-hd`) is attached to.

Reply:
`Want me to remove the id=scsi?`

Only if the ID is unused. I'm not 100% sure what the ID is used for in QEMU right now

Reply:
I removed the explicit naming on both #1590 and #1589